src/View.elm

```diff
@@ -0,0 +1,178 @@
+module View exposing (viewStyled)
+
+import Css exposing (..)
+import Dict exposing (Dict)
+import Html.Styled exposing (..)
+import Html.Styled.Attributes exposing (css, placeholder, value)
+import Html.Styled.Events exposing (onClick, onInput)
+import Model exposing (..)
+import RemoteData exposing (RemoteData(..))
+
+
+viewStyled : Model -> Html Msg
+viewStyled model =
+    div [] [ viewRemoteData "list of users" model.users (viewUsers model) ]
+
+
+viewRemoteData : String -> RemoteData (List String) a -> (a -> Html Msg) -> Html Msg
+viewRemoteData elementName rdata viewSuccess =
+    case rdata of
+        NotAsked ->
+            text ("Initializing " ++ elementName)
+
+        Loading ->
+            text ("Loading " ++ elementName)
+
+        Failure errors ->
+            text ("Error loading " ++ elementName ++ ": " ++ String.join ", " errors)
+
+        Success x ->
+            viewSuccess x
+
+
+viewUsers : Model -> Dict Int User -> Html Msg
+viewUsers ({ underEdit } as model) users =
+    let
+        theUsers =
+            Dict.toList users
+
+        medallions =
+            (List.take 3 theUsers |> List.map (viewUser medallion underEdit)) ++ [ viewCreateUserMedallion underEdit ]
+
+        medallionsArea =
+            div
+                [ css
+                    [ property "display" "grid"
+                    , property "grid-template-columns" "repeat(auto-fill, minmax(min(100%, 180px), 1fr))"
+                    , property "grid-gap" "8px"
+                    , padding (px 8)
+                    , borderBottom2 (px 1) solid
+                    ]
+                ]
+                medallions
+
+        rows =
+            List.drop 3 theUsers |> List.map (viewUser row underEdit)
```

Comment:
`medallions` and `rows`: interesting choice of parameterizing the `viewUser` - although I don't feel like the app-specific `view` code (so, excluding eg. `viewRemoteData`) needs to have this level of flexibility... 🤔

Reply:
It's more about code reuse than flexibility. But it's more a coincidence that the two user representations can have the same internal layout than a design decision.